### PR TITLE
More fixes for buildbot.net buildslave

### PR DESCRIPTION
**Pull request by seankelly**
Handful of commits to install and set everything needed to get the buildbot.net buildslave functional.

Notable:
* Added two roles: one for managing SSH private keys and another for the known_hosts file.
* Put the private key buildbot.net uses in secrets.yml. This was generated by hand.
* Put SSH host keys for ftp.int.buildbot.net:2200 in `group_vars/all`. I don't know if anything else needs to use SSH communication, but thought this would be the best location for them.
* Added more packages that are needed to run Buildbot.

All of the commits except c785d13 have been tested.

### CODE REVIEW

**Comment by seankelly** [comment]
To fix I think everything, also need buildbot/metabbotcfg#24 merged.

File: jail-ftp.yml [hunk 1]
```diff
@@ -23,7 +23,8 @@
   - role: user
     user_id: buildbot
     user_name: Buildbot Uploader
-    authorized_key: >
+    authorized_key: |
       ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC7YoNGKRZJxVbvxQpGBX1FuhWC0Nq2+fe+wVD16cgaOk4iDjeKdQ4eZACIlcMi5REXfTO+e9Hzmr85DUmy+BAKx0PqzLj6BhmnZnoQbQe3WAsmWSIuUSqKqilM4RnkDUWrDvhdyMGIH4ELfrebJt5ZnNLbOTk6BAQiygF/1xNeYPvcjxfUG1NXUHlKJ9qnf1cph7Pw6YOe8SF8kJTxwDiSJSG0aGsfas4zLVgmcPHdjsFRuOFOxFlVpFKzu3toaZyXPSeTOAaoaIHkfaEahBExXyd7b8L/0fcFumWK6qwIA5dFDWGkIaEMBdX6/giV+gvz9UJQ75gxVtn0y8MbFEMD9iduGbIX5FSOFNmbEkNubW5M74G7igMtqk+dUbVSMOoy4NO2vPnXH0iBMPkkzPEmAro3/q/J5QTEbIB39H99dbGnOR1iwbBhCqVJERAAECteoHsXKHsNhL9IYqDp2KKPVHmxZybqtJo4SKSzNvjfK50ojYcIAAVlBsFZ+9p0Aj8CADA3L0Ont7RPs9Dcoad1la9kkC1v6Bol5O1huRM8qjXRPOT1W01JH+UvM8r1EKNKRgV/uVr+Zl1Ln84GvPMX4cdz+ky2L5mlE9w6RonBqJnJg16i7eQXCE6iwOjQ2XHK+TBeoQDGHmww+ACUmVx9NR+I4Q9cWMjteLx3Q+YMtQ== buildbot@ftp.buildbot.net
```
Comment (djmitche) [inline jail-ftp.yml]: I think this key is for the old buildbot.net buildslave.  So it should probably be removed.

**Comment by djmitche** [comment]
Looks good.  I'll let you merge when you're around to see the consequences.

**Comment by seankelly** [comment]
I don't have write permissions to the repository.

**Comment by tardyp** [comment]
thanks!!

Le lun. 4 mai 2015 à 03:45, Dustin J. Mitchell <notifications@github.com> a
écrit :

**Pull request merged**